examples/TID_model_custom_property/Modelling_TID_TNID_example_notebook.ipynb

```diff
@@ -0,0 +1,471 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    for x in bins_count_g:
</pre><p>for else? I don't think this code works as you intend, it will simply execute the "else" case after the for loop</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>